versions.json

```diff
@@ -1,9 +1,9 @@
 {
     "messages": {
-        "version": "v1.0.0"
+        "version": "v1.3.0"
```

Comment:
why does this version jump from 1.0 to 1.3?

Reply:
NVM realized it's because there was not a full SDK version update when these packages were last updated

Reply:
Because there was a 1.2.0 that was released for the message package but not included in the SDK. The sub packages dont always warrant a new SDK.